src/e2e/scala/temple/SimpleEndToEndTest.scala

```diff
@@ -0,0 +1,50 @@
+package temple
+
+import java.io.IOException
+import java.nio.file.attribute.BasicFileAttributes
+import java.nio.file.{FileVisitResult, Files, Path, Paths, SimpleFileVisitor}
+
+import org.scalatest.{FlatSpec, Matchers}
+
+class SimpleEndToEndTest extends FlatSpec with Matchers {
+  behavior of "Temple"
+
+  private def removeDirectory(path: Path): Unit =
+    Files.walkFileTree(
```

Comment:
Is there no simple way to recursively delete a directory? Is this your own code, or can we cite it?